test/e2e/pkg/manifest.go

```diff
@@ -130,6 +130,11 @@ type ManifestNode struct {
 	// For more information, look at the readme in the maverick folder.
 	// A list of all behaviors can be found in ../maverick/consensus/behavior.go
 	Misbehaviors map[string]string `toml:"misbehaviors"`
+
+	// Log level sets the log level of the specific node i.e. "consensus:info,*:error".
+	// This is helpful when debugging a specific problem. Note that setting this
+	// will override any log level flag.
+	LogLevel string `toml:"log_level"`
```

Comment:
It's odd to me that we can set per-node log levels in the manifest, but have to use a flag for the global log level. Would be nice if global log level could be set in manifest as well (as well as via flag, where the flag overrides).

Reply:
I originally thought that just having a flag could cover global testnet logs and although perhaps arduous, one could always add the individual log level to every node in the manifest to get the same global behavior. But it shouldn't be too difficult to allow someone to set the global log level in the manifest (which can then get overwritten if a flag is set)

Reply:
Yeah, I wonder if it might be easier to just use the manifest for this and drop the flag. While the flag could be convenient, it can be pretty fiddly to handle all of the combinations, especially considering the setup/run issue I listed below, and I'm not sure if it's worth the trouble. Your call though.

Reply:
Ok I think I'm just going to drop the flag. It's not difficult at all to just add the `log_level` field to the manifest